sycl/test-e2e/bindless_images/vulkan_interop/mipmaps.cpp

```diff
@@ -289,7 +293,8 @@ bool run_test(sycl::range<NDims> dims, sycl::range<NDims> localSize,
                 std::max(height >> i, (uint32_t)1) *
                 std::max(depth >> i, (uint32_t)1));
     for (int j = 0; j < mipElems; ++j) {
-      inputStagingData[j + offset] = initVector<DType, NChannels>(j);
+      inputStagingData[j + offset] =
+          bindless_helpers::init_vector<DType, NChannels>(j);
```

Comment:
You're interchangeably using `init_vector` vs `bindless_helpers::init_vector` in this test file.

Between test files you're interchangeably using `using` and consequently whether you have `bindless_helpers::` in each function call.

IMO, I'd prefer it if you removed `using` and were explicit but that can be decided upon. However, you should definitely be consistent within the files and each test file should follow the same rules.

Reply:
adopted use of `bindless_helpers::` for `init_vector` and `using` for `equal_vec` across vulkan interop test files. I didn't like using `namespace::foo` inside if conditions that's why I kept `using bindless_helpers::equal_vec;`.